src/mapping/config/MappingConfiguration.cpp

```diff
@@ -217,6 +217,8 @@ MappingConfiguration::MappingConfiguration(
   auto attrPolynomial = makeXMLAttribute(ATTR_POLYNOMIAL, POLYNOMIAL_SEPARATE)
                             .setDocumentation("Toggles use of the global polynomial")
                             .setOptions({POLYNOMIAL_ON, POLYNOMIAL_OFF, POLYNOMIAL_SEPARATE});
+  auto attrCrossValidation = makeXMLAttribute(ATTR_CROSS_VALIDATION, false)
+                                 .setDocumentation("Computes and logs a cross-validation error using the coupling data and the mapping matrix. This can be useful for tuning the basis function. Note that this might become computationally expensive.");
```

Comment:
```suggestion
                                 .setDocumentation("Computes and logs a cross-validation error using coupling data and the mapping matrix. This can be useful for tuning the basis function. Note that this might become computationally expensive.");
```

To make clearer that the mapping matrix is static and the data dynamic.

Reply:
Could we give an estimate on the "computationally expensive"? 5x higher than the mapping without cross validation or similar?